src/node-jspm-jasmine.js

```diff
@@ -366,8 +366,10 @@ function importTestFiles(SystemJS, jasmine, specDir, specFiles, coverage, errCal
 					}
 				})
 				.catch((ex) => {
-					// remove temporary directory
-					rimraf.sync(coverage.tempDirectory);
+					if (coverage && coverage.tempDirectory) {
```

Comment:
`coverage` is undefined when the tests are being run without coverage, which was causing an error before errCallback was being called (which ensures that everything is logged to the console correctly).